nodestream/cli/commands/make_migrations.py

```diff
@@ -0,0 +1,25 @@
+from cleo.helpers import option
+
+from ..operations import GenerateMigration
+from .nodestream_command import NodestreamCommand
+from .shared_options import PROJECT_FILE_OPTION
+
+
+class MakeMigration(NodestreamCommand):
+    name = "migrations make"
+    description = "Generate a migration for the current project."
+    options = [
+        PROJECT_FILE_OPTION,
+        option(
+            "dry-run",
+            description="Don't generate the migration file, just output the detected changes.",
+            flag=True,
+        ),
+    ]
+
+    async def handle_async(self):
```

Comment:
Some logging or error handling could be helpful

Reply:
Some context in the surrounding code: 

The CLI framework accepts and pretty prints and describes exceptions that are thrown. You can customize the level of verbosity that the error information includes by passing `-v`, `-vv`, and `-vvv`. 

Are you suggesting that we add error handling beyond what is included in the description above?

Reply:
I think it's fine